src/custom-operations/apply-traits.ts

```diff
@@ -0,0 +1,60 @@
+import { JSONPath } from 'jsonpath-plus';
+
+import { xParserOriginalTraits } from '../constants';
+import { mergePatch } from '../utils';
+
+const v2TraitPaths = [
```

Comment:
`v2` means `v2` of the spec here? If so, shall we extract the paths to their own files so we can do the same for `v3` spec ?

Reply:
We will have only 2 custom operations and I don't think so that we should split them to separate files. We can do it later, if we will have more logic for each version.